scripts/scil_visualize_gradients.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Vizualisation for sampling schemes.
+Only supports .bvec/.bval and .b (MRtrix).
+"""
+
+import argparse
+import logging
+import numpy as np
+import os
+
+from scilpy.utils.bvec_bval_tools import identify_shells
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_gradients_filenames_valid,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.viz.sampling_scheme import (build_ms_from_shell_idx,
+                                        plot_each_shell,
+                                        plot_proj_shell)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=__doc__)
+
+    p.add_argument(
+        'scheme_file', metavar='scheme_file', nargs='+',
+        help='Sampling scheme filename. (only accepts .bvecs and .bvals '
+             'or .b).')
```

Comment:
This help does not match the docstring above. 
Moreover, it is not clear. It works for a .b file, that's fine.
But for the fsl format, it requires both files .bval and .bvec. In any order, it works. But .bvals/.bvecs fails...

Please clarify.

Reply:
Done